SDK/AppCenter/Microsoft.AppCenter.Shared/WrapperSdkInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is Shared code, meaning you want to the ability to wrap Xamarin SDK in another layer on top of Xamarin (I also don't see any Xamarin change).

Was the intent to put it in Windows.Shared project instead? That would be better if it applies only to Windows platforms.